### ISSUE TITLE
Rectangle placeholder without className sets class to undefined

### ISSUE THREAD

**Issue by dwightwatson** [buildo/react-placeholder#24]
A minor issue, but if a `className` is not provided when using the rectangle placeholder, one of the classes applied will be `undefined.

![image](https://cloud.githubusercontent.com/assets/1100408/25567766/014c050e-2ded-11e7-877b-0b6bc573babf.png)

An easy fix might be to build up the classes with an array, filter the empty ones and join it up. Happy to PR this if it's helpful.

```js
const classes = ['rect-shape', className]
  .filter(c => c)
  .join(' ')
```

**Comment by gabro** [comment]
Oops, we probably introduced this when we stripped away the classnames dependency.
Thanks for reporting, a PR would be surely appreciated!

**Comment by FrancescoCioria** [comment]
Closed by #25
